.github/workflows/ci_cd.yml

```diff
@@ -91,8 +103,11 @@ jobs:
       - name: Release
         uses: softprops/action-gh-release@v1
         with:
+          prerelease: true
```

Comment:
Presumably these three additions will be removed before merge.

Reply:
Indeed